apps/api/src/app/content-templates/usecases/compile-template/compile-template.spec.ts

```diff
@@ -61,10 +62,29 @@ describe('Compile Template', function () {
       })
     );
 
+    it('should allow the user to specify handlebars helpers'),
+      async function () {
+        const result = await useCase.execute(
+          CompileTemplateCommand.create({
+            templateId: 'custom',
+            data: {
+              branding: {
+                color: '#e7e7e7e9',
+              },
+              message: 'hello world',
+            },
+            customTemplate: '<div>{{titlecase message}}</div>',
+          })
+        );
+      };
+
     expect(result).to.contain('Hello TESTTTT content');
     expect(result).to.not.contain('{{#each blocks}}');
     expect(result).to.not.contains('ff6f61');
     expect(result).to.contain('#e7e7e7e9');
     expect(result).to.contain('Button content of text');
+    expect(result).to.contain('{{titlecase message}}');
+    expect(result).to.contain('{{lowercase message}}');
+    expect(result).to.contain('{{uppercase message}}');
```

Comment:
You did not use those two in the customTemplate :) You should use the handlebars and their helpers in the customTemplate. And in the expect(..) you check that you got the result you expected to get. 
Lets say you set:

`
customTemplate: <div>{{titlecase message}} and {{uppercase message}}</div>
`

The check would be: 

`
expect(result).to.equal('<div>Hello World and HELLO WORLD</div>')
`